front/app/modules/commercial/analytics/admin/hooks/useVisitorsData/index.ts

```diff
@@ -106,12 +108,12 @@ const query = ({
   };
 };
 
-export default function useVisitorsData({
-  projectId,
-  startAtMoment,
-  endAtMoment,
-  resolution,
-}: QueryParameters) {
+export default function useVisitorsData(
+  formatMessage: InjectedIntlProps['intl']['formatMessage'],
+  { projectId, startAtMoment, endAtMoment, resolution }: QueryParameters
+) {
```

Comment:
Added this `deducedResolution` just to make sure that we're always using the correct resolution. I am not sure, but it might be possible that the resolution changes already while the data returned by the observable still has the old resolution, because the request hasn't been processed or something. That would break the logic. This way, you never have to worry about that. Also the `deducedResolution` is used in the card component to decide what copy to display- this way, we only show the new copy once the request is finished